src/lib/Microsoft.Health.Fhir.Ingest.Template/Microsoft.Health.Fhir.Ingest.Template.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <CodeAnalysisRuleSet>..\..\..\CustomAnalysisRules.ruleset</CodeAnalysisRuleSet>
     <HighEntropyVA>true</HighEntropyVA>
     <RootNamespace>Microsoft.Health.Fhir.Ingest.Template</RootNamespace>
-    <LangVersion>7.3</LangVersion>
```

Comment:
><LangVersion>7.3</LangVersion> [](http://example.com/codeflow?start=4&length=30)

Please add this back.  You can update it to <LangVersion>8.0</LangVersion> which it appears you need with the latest changes but without builds can fail.  I.E. the latest fails to build on my dev machine.

Reply:
I actually added that in earlier this morning. Strange its not showing up in github... will try refreshing my browser

Reply:
There it is. Must have been some Github weirdness.